Don't use CPP/CXXCPP from environment after bug 1299919. r=glandium, a=jcristau

diff --git build/buildconfig.py build/buildconfig.py
index bb12114b2f4a..e73903a1a981 100644
--- mozilla/build/buildconfig.py
+++ mozilla/build/buildconfig.py
@@ -16,5 +16,5 @@ for var in ('topsrcdir', 'topobjdir', 'defines', 'non_global_defines',
 substs = dict(substs)
 
 for var in os.environ:
-    if var != 'SHELL' and var in substs:
+    if var not in ('CPP', 'CXXCPP', 'SHELL') and var in substs:
         substs[var] = os.environ[var]
